src/pages/Feedback.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import GameHeader from '../components/GameHeader';
+
+
+class Feedback extends Component {
+  render() {
+    const { player } = this.props;
+    const { assertions, score } = player;
+    return (
+      <div>
+        <GameHeader />
+        <p data-testid="feedback-text">{(assertions >= 3 ?
+          'Mandou bem!' : 'Podia ser melhor...')}</p>
+
+        <p>Você acertou <span data-testid="feedback-total-question">
+          {assertions}</span> questões</p>
+        <p>Um total de <span data-testid="feedback-total-score">{score}</span> pontos</p>
+        <Link to="/rank">
```

Comment:
```suggestion
        <Link to="/ranking">
```
nos mudamod tudo que tava com rank para ranking, ai tem que mudar o nome do arquivo que tem isso e todas as rotas e inports que usam isso hahahaa

Reply:
Boaaaa Ronam!!! :nerd_face: